src/entity/Living.php

```diff
@@ -188,6 +188,13 @@ public function setHealth(float $amount) : void{
 		}
 	}
 
+	public function getGravity() : float{
+		if($this->effectManager->has(VanillaEffects::SLOW_FALLING())){
+			return $this->effectManager->get(VanillaEffects::SLOW_FALLING())->getType()->getGravity();
```

Comment:
This can be reduced to `VanillaEffects::SLOW_FALLING()->getGravity()`.

Reply:
ok